src/repo.rs

```diff
@@ -1088,6 +1088,21 @@ impl Repository {
         }
     }
 
+    /// Get the blame for a single file.
+    pub fn blame_file<'a>(&self, path: &Path) -> Result<Blame<'a>, Error> {
```

Comment:
I think this lifetime parameter means that the returned `Blame` is connected to any lifetime? You should be fine just omitting this entirely (the lifetime parameter)